flake.nix

```diff
@@ -32,15 +32,46 @@
           z3
           cvc5
           git
-          foundry.defaultPackage.${system}
+          # foundry.defaultPackage.${system}
```

Comment:
Is there a reason why this needs to be removed?

Reply:
Updated with conditional code. It failed on `aarch64-darwin`.